sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1477,12 +1477,12 @@ class Analyzer(
           resolveSubQuery(s, plans)(ScalarSubquery(_, _, exprId))
         case e @ Exists(sub, _, exprId) if !sub.resolved =>
           resolveSubQuery(e, plans)(Exists(_, _, exprId))
-        case InSubquery(values, l @ ListQuery(_, _, exprId, _))
+        case p @SubqueryPredicate(values, comparison, l @ ListQuery(_, _, exprId, _))
```

Comment:
nit: need a space after `@`. Also, could you check other code style along with the other formats? I found some style errors.

Reply:
fixed that, thanks.